### PR TITLE
Renamed Completable#finallyDo to #doAfterTerminate

### PR DESCRIPTION
**Pull request by stealthcode**
Renames as part of #3434 for the recently added Completable.

### CODE REVIEW

File: src/test/java/rx/CompletableTest.java [hunk 1]
```diff
@@ -1926,7 +1926,7 @@ public void onCompleted() {
     public void finallyDoWithError() {
```
Comment (akarnokd) [inline src/test/java/rx/CompletableTest.java]: Maybe the test names should be changed as well.

**Comment by artem-zinnatullin** [comment]
:+1:

**Comment by akarnokd** [comment]
:+1:

**Comment by stevegury** [comment]
:+1:

**Pull request merged**